tomcat8-nojsvc@.service

```diff
@@ -0,0 +1,36 @@
+# Systemd unit file for tomcat instances.
+#
+# To create clones of this service:
+# 0. systemctl enable tomcat8-nojsvc@name.service
+# 1. create catalina.base directory structure in
+#    /var/lib/tomcats/name
+# 2. set catalina base inside /etc/sysconfig/tomcat8@<name>
+
+[Unit]
+Description=Apache Tomcat 8 Web Application Container (No Jsvc)
+After=syslog.target network.target
+
+[Service]
+Type=forking
```

Comment:
With `Type=simple` PID file is not necessary ;)

Can we get rid of `Type=forking` ?

Reply:
The new version of unit template is of type `Simple`